imagelib/admin/thumbnailbuilder.php

```diff
@@ -72,6 +72,7 @@ function resetRebuildForm(f){
 	</div>
 	<!-- This is inner text! -->
 	<div id="innertext">
+		<h1 class="page-heading"><?= $LANG['THUMB_BUILDER']; ?></h1>
```

Comment:
This is redundant with the h2, maybe just move the h2 to be h1 (Thumbnail Maintenance Tool)

Reply:
Fixed in 9fc83a2ab.